### PR TITLE
feat: google auth

### PR DESCRIPTION
**Pull request by zalevskyi**
[task description](https://trello.com/c/NnQYmNfp/111-add-an-ability-to-login-using-google)

### CODE REVIEW

File: backend/src/services/auth.service.ts [hunk 1]
```diff
@@ -24,6 +24,8 @@ import { ISetPassword } from '../common/interfaces/auth/set-password.interface';
 import jwt from 'jsonwebtoken';
 import { IUpdatePasswordAndFullName } from 'infostack-shared';
```
Comment (ghost) [inline backend/src/services/auth.service.ts]: No imports from infostack-shared

File: frontend/package.json [hunk 2]
```diff
@@ -25,6 +25,7 @@
     "react-bootstrap": "2.0.0-beta.4",
     "react-cookie": "4.0.3",
     "react-dom": "17.0.2",
+    "react-google-button": "^0.7.2",
```
Comment (ghost) [inline frontend/package.json]: B17. The versions of dependencies used are fixed in package.json.

File: frontend/package.json [hunk 3]
```diff
@@ -34,8 +35,8 @@
     "react-scroll": "1.8.3",
     "react-select": "4.3.1",
     "react-toastify": "7.0.4",
-    "remark-slug": "7.0.0",
     "remark-gfm": "1.0.0",
+    "remark-slug": "7.0.0",
```
Comment (ghost) [inline frontend/package.json]: Unnecessary change (view string 37)

File: frontend/src/components/login-google/login-google.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import { useAppDispatch, useHistory, useEffect } from 'hooks/hooks';
+import { authActions } from 'store/auth';
+import { AppRoute } from 'common/enums/enums';
+import { getAllowedClasses } from 'helpers/dom/dom';
```
Comment (ghost) [inline frontend/src/components/login-google/login-google.tsx]: Maybe it wil be better to export from helpers/helpers

**Review by ghost — APPROVED** [review]
It will be better to create a folder login-google not in the components root, but inside the auth folder

File: frontend/package.json (hunk 3 above)
Comment (zalevskyi) [reply frontend/package.json]: Unfortunately npm install does not discuss changes it makes )

**Review by aleksandr-kuharenko — APPROVED** [review]
